### DISCUSSION TITLE
disable local storage of (unencrypted) content data when syncing

### DISCUSSION THREAD

**Discussion by eggenbef** [AsamK/signal-cli#1644] [General]
hi there
just wonder whether there is an option when calling "signal-cli receive" to prevent potentially confidential data from being stored locally. it seems that the receive call will be required regularly to keep the signal-cli device linked to the account. that's why i am running this command on a regular basis as part of a script but noticed that with every such call i get all data (specifically attachments) of conversations which happened since the last receive call unencrypted stored on my local drive. the workaround i've got in place now is a script which deletes this data whenever the sync has finished but an option to instruct signal-cli **not** to save any content of past conversations received might be more straightforward.

**Comment by AsamK** [comment]
You can use `signal-cli receive --ignore-attachments` to prevent download and saving of attachments
